src/reducers/sources.js

```diff
@@ -312,7 +330,11 @@ export function getPrettySource(state: OuterState, id: string) {
     return;
   }
 
-  return getSourceByURL(state, getPrettySourceURL(source.url));
+  return getSourceByURL(
+    state,
+    getPrettySourceURL(source.url),
+    isOriginalId(source.id)
+  );
```

Comment:
The way this is being used at present, it appears, is to check if a pretty source for a given generated source exists, so I think it makes sense.

I'm very glad you asked this, however, because I think this *could be* what's causing the lone test to fail; I think for pretty-printed tabs the `isOriginalId` may not be correct which might cause a that failure.